### ISSUE TITLE
[P] T072: Define Prometheus alert rules

### ISSUE THREAD

**Issue by kwcantrell** [kwcantrell/ophir-training-app#62]
Define Prometheus alert rules for: NaN loss (training_loss = NaN), GPU OOM (job event count > 0 for oom type), job stagnation (loss unchanged for 5 min) in `infra/prometheus/alerts.yml`.
